eisbuk-admin/src/components/atoms/OrganizationSettings/OrganizationSettings.tsx

```diff
@@ -0,0 +1,396 @@
+import React, { useState } from "react";
+import { OrganizationData } from "eisbuk-shared/dist";
+// import { useTranslation } from "react-i18next";
+
+// import { useDispatch } from "react-redux";
+import { Formik, Field, Form, useField } from "formik";
+
+import FormControl from "@material-ui/core/FormControl";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import Cancel from "@material-ui/icons/Cancel";
+
+import makeStyles from "@material-ui/core/styles/makeStyles";
+
+import { Divider, IconButton, Typography } from "@material-ui/core";
+import clsx from "clsx";
+import { useSelector } from "react-redux";
+import { getLocalAuth } from "@/store/selectors/auth";
+
+interface Props {
+  organization: OrganizationData;
+}
+const OrganizationSettings: React.FC<Props> = ({ organization }) => {
+  const classes = useStyles();
+  const userAuthInfo = useSelector(getLocalAuth);
+
+  const [enableEdit, setEnableEdit] = useState(false);
+  const handleSubmit = (orgData: OrganizationData) => {
+    console.log({ orgData });
+    // useDispatch
+    setEnableEdit(false);
+  };
+
+  const handleEnableEdit = () => {
+    setEnableEdit(!enableEdit);
+  };
+  const currentUser = userAuthInfo?.email || "";
+  return (
+    <>
+      <div className={classes.title}>
+        <Typography variant="h4">Organization Settings</Typography>
+      </div>
+      <div className={classes.content}>
+        {/* <Typography variant="h4">Organization Settings</Typography> */}
+
+        <Formik
+          {...{ initialValues: { ...organization } }}
+          onSubmit={handleSubmit}
+          validateOnChange={false}
+        >
+          {({ errors, isSubmitting, isValidating }) => (
+            <Form className={classes.form}>
+              <FormControl component="fieldset">
+                <h5 className={classes.intervalsTitle}>Admins</h5>
+                <AdminsField
+                  enableEdit={enableEdit}
+                  currentUser={currentUser}
+                />
+                <Divider />
+                <h5 className={classes.intervalsTitle}>SMTP</h5>
+
+                <div className={classes.fieldSection}>
+                  <Field
+                    label="SMTP Server"
+                    name="SMTPServer"
+                    className={classes.field}
+                    as={TextField}
+                    aria-label="SMTPServer"
+                    placeholder="SMTP server here"
+                    variant="outlined"
+                    value={organization.SMTPServer}
```

Comment:
This `value` will stay static. You either want to get values from `Formik`'s render function `({errors, isSubmitting, values}) => <Field value={values.<fieldName>}>`. This way you would need to manually set value for each field using `onChange` handler.
Better approach would be to avoid this complexity and let the `Field` component control value displaying and updateing as it is programmed to do just that, the only input it needs to connect to Formik context is the right `name` prop (as we've discussed)